linera-rpc/src/grpc/client.rs

```diff
@@ -279,7 +279,13 @@ impl ValidatorNode for GrpcClient {
                     true
                 })
             })
-            .filter_map(|result| future::ready(result.ok()));
+            .filter_map(|result| {
```

Comment:
Maybe not in this PR, but it feels like we should at least `debug` log any errors :thinking:

Reply:
done